navigation.mdx

```diff
@@ -584,6 +584,34 @@ Style pages with breadcrumbs to display the full navigation path at the top of p
 }
 ```
 
+## Interaction Configuration
+
+Control how users interact with navigation elements using the `interaction` property in your `docs.json`.
+
+### Auto Navigation on Expand
+
+Different themes have different default behavior when clicking on navigation groups. You can override this behavior using the `drilldown` option:
+
+```json
+{
+  "interaction": {
+    "drilldown": true  // Force navigation to first page when a user expands a dropdown
+  }
+}
+```
+
+```json
+{
+  "interaction": {
+    "drilldown": false // Never navigate, only expand/collapse
+  }
+}
+```
```

Comment:
we should also add this property in `/settings` page

Reply:
Sounds good. I can move it there.